crates/bdk/Cargo.toml

```diff
@@ -22,12 +22,13 @@ serde_json = { version = "^1.0" }
 bdk_chain = { path = "../chain", version = "0.6.0", features = ["miniscript", "serde"], default-features = false }
 
 # Optional dependencies
-hwi = { version = "0.7.0", optional = true, features = [ "miniscript"] }
+hwi = { version = "0.7.0", optional = true, features = ["miniscript"] }
 bip39 = { version = "1.0.1", optional = true }
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 getrandom = "0.2"
 js-sys = "0.3"
+bitcoin = { version = "0.30.0", features = ["no-std"] }
```

Comment:
missing WASM dep that somehow cargo does not cover and is not vendored in the Nix derivation... 🤷🏻 
Check: https://github.com/ipetkov/crane/issues/439